include/odp/api/spec/traffic_mngr.h

```diff
@@ -189,6 +189,47 @@ extern "C" {
  * tree/hierarchy of nodes.
  */
 
+/**
+ * TM node stats capability
+ */
+typedef struct odp_tm_node_stats_capability_t {
+	/** Supported counters */
+	union {
+		/** Statistics counters supported in bit field struct */
+		struct {
+			/** @see odp_tm_node_stats_t::dropped_pkts */
+			uint64_t dropped_pkts;
```

Comment:
We use `discard` in all other packet statistics. Also, I think we can spare a few letters and just use e.g. `discarded_packets`.

Reply:
I wonder if "discarded" is really better than "dropped" in the TM API. The term drop is often used in traffic management context and also in the ODP TM API spec, which discusses drop probabilities when (W)RED is used, RED versus tail drop algorithm and so forth.

Reply:
We ended up naming these counters `discard_octets`and `discard_packets` in #1305.